src/project-scanner.ts

```diff
@@ -12,14 +12,31 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-import {Analysis, Document} from 'polymer-analyzer';
+import {Analysis} from 'polymer-analyzer';
 
 import {ConversionSettings} from './conversion-settings';
-import {DeleteFileScanResult, DocumentConverter, HtmlDocumentScanResult, JsModuleScanResult} from './document-converter';
+import {DeleteFileScanResult, HtmlDocumentScanResult, JsModuleScanResult, ScanResult} from './document-converter';
 import {JsExport} from './js-module';
+import {PackageScanner} from './package-scanner';
 import {OriginalDocumentUrl} from './urls/types';
 import {UrlHandler} from './urls/url-handler';
 
+function exportsMapToObject(exportsMap: Map<string, JsExport>) {
+  const exportsObject: any = {};
+  for (const [k, v] of exportsMap) {
+    exportsObject[k] = { name: v.name, url: v.url };
+  }
+  return exportsObject;
+}
+
+function filesMapToObject(filesMap: Map<OriginalDocumentUrl, ScanResult>) {
+  const filesObject: any = {};
+  for (const [k, v] of filesMap) {
+    filesObject[k] = (<any>v).convertedUrl || null;
```

Comment:
why the `any` cast? If it's really needed, use `as` syntax

Reply:
This is also removed in the next PR